src/org/opendatakit/briefcase/reused/UncheckedFiles.java

```diff
@@ -302,6 +302,8 @@ public static InputStream newInputStream(Path path, OpenOption... options) {
   }
 
   public static void closeInputStream(InputStream inputStream) {
+    if (inputStream == null)
```

Comment:
This feels wrong. What's the rationale behind supporting a `null` `inputStream` arg?

Reply:
Since it's a static method that could be called in other places, on the off chance that an inputStream is passed and is null, right now, it's not handling the null pointer exception.

The other option could be to update the catch to catch both `catch(NullPointerException | IOException error)`, and throw the UncheckedException.

Otherwise, if you believe it doesn't need to be handled, then I can remove it.